package-scripts.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     },
     validate: {
       description: 'This runs several scripts to make sure things look good before committing or on clean install',
-      script: `p-s -p ${validate.join(',')} && p-s check-coverage`,
+      script: `dist/bin/p-s.js -p ${validate.join(',')} && p-s check-coverage`,
```

Comment:
This is part of the temporary change. I'll change it back when we get the fix released.